apps/emqx/include/logger.hrl

```diff
@@ -69,6 +69,8 @@
                 ok
         end).
 
+-define(TRACE(Action, Meta, Msg), emqx_trace:log(Action, Meta, Msg)).
```

Comment:
Action-> event?

Reply:
And it’s maybe redundant information with Msg
Maybe enough to just use a readable static Msg instead of using two fields.